### ISSUE TITLE
Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION

### ISSUE THREAD

**Issue by fsaxen** [tensorflow/tensorflow#17747]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.6.0
- **Python version**: 3.5.2
- **Bazel version (if compiling from source)**: N/A
- **GCC/Compiler version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 9.0.176.2 / cudnn7.0.5
- **GPU model and memory**: GTX 1080 (8GB)
- **Exact command to reproduce**:
```
import numpy as np
import tensorflow as tf

batch_size = 64
images = tf.random_normal(shape=[batch_size, 32, 32, 3], dtype=tf.float32)
angles = tf.random_uniform([batch_size], -0.5, 0.5)
images = tf.contrib.image.rotate(images, angles)

with tf.Session() as sess:
    _ = sess.run(images)
```
### Any Idea why this small example produces the following error?
**ERROR:**
*2018-03-16 18:27:24.292665: E C:\tf_jenkins\workspace\rel-win\M\windows-gpu\PY\35\tensorflow\stream_executor\cuda\cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION*
*2018-03-16 18:27:24.292700: E C:\tf_jenkins\workspace\rel-win\M\windows-gpu\PY\35\tensorflow\stream_executor\cuda\cuda_driver.cc:1110] could not synchronize on CUDA context: CUDA_ERROR_ILLEGAL_INSTRUCTION ::*
*2018-03-16 18:27:24.296409: F C:\tf_jenkins\workspace\rel-win\M\windows-gpu\PY\35\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:203] Unexpected Event status: 1*

**UPDATE:**
I think issue #17485 is very similar

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version

**Comment by gzzzzzzz** [comment]
tf.contrib.image.rotate(images, angles) crashed on Win10，python3.6，tensorflow1.6，CUDA9，cudnn7.0，1080Ti。
Error Code：failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-03-17 22:26:12.688690: F C:\tf_jenkins\workspace\rel-win\M\windows-gpu\PY\36\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:203] Unexpected Event status: 1

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by aligokalppeker** [comment]
Same issue is valid for my environment also; Windows 10 x64, Tensorflow 1.7.0, CUDA 9, cuDNN 7

2018-04-29 21:11:23.422100: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-04-29 21:11:23.422100: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION
2018-04-29 21:11:23.422361: F T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:203] Unexpected Event status: 1
2018-04-29 21:11:23.422608: F T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:203] Unexpected Event status: 1

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by lixiang-ucas** [comment]
got same issue using tf1.4.0,cuda8.0,cudnn6.0.21

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by AloshkaD** [comment]
I recreated a fresh conda env with cdnn, tensorflow gpu1.8, and keras gpu 2.1 installed from anaconda. Before that I downgraded my cuda from 9.2 to 9.0

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by manikaaggarwal** [comment]
I also get this issue, in between of the training.

**Comment by aselle** [comment]
Do you know about windows CUDA support,?

**Comment by WhiteCipher** [comment]
same error! 
 Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_ADDRESS

**Comment by Henrystarstar** [comment]
same error~ Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_INSTRUCTION

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @zheng-xq: It has been 30 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by maidaly** [comment]
I also have the same error . CUDA_ERROR_ILLEGAL_INSTRUCTION. we just need to know which instruction caused this error as the gpu works fine else and the code works fine with if we change the device to be cpu instead.

**Comment by gogobd** [comment]
I've got the same issue with Windows 10, tensorflow 1.8.0, cudatoolkit 9.0 (anaconda), cudnn 7.1.4 (anaconda) on a GTX 970 (msi), any help would be very appreciated...

**Comment by ymodak** [comment]
@fsaxen Is this still an issue? @gogobd Can you please test it against latest TensorFlow version and post your findings?

**Comment by ymodak** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by gogobd** [comment]
I updated to 1.11.0 and the problem is gone. Thank you!

**Comment by zhangjunhust** [comment]
> same error!
> Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_ADDRESS

Have you figured it out?
